lib/src/androidTest/java/com/couchbase/lite/internal/core/C4MutableFleeceTest.java

```diff
@@ -93,9 +104,14 @@ static void verifyDictIterator(Map<String, Object> dict) {
     }
 
     static String fleece2JSON(AllocSlice fleece) {
-        FLValue v = FLValue.fromData(fleece);
-        if (v == null) { return "INVALID_FLEECE"; }
-        return v.toJSON5();
+        try {
+            FLValue v = FLValue.fromData(fleece);
+            if (v == null) { return "INVALID_FLEECE"; }
+            return v.toJSON5();
+        }
+        finally {
+            if (fleece != null) fleece.free();
```

Comment:
may be add a comment, describing about this release of the fleece object. So caller don't need to worry about it.